mllib/src/main/scala/org/apache/spark/ml/Predictor.scala

```diff
@@ -49,10 +50,7 @@ private[ml] trait PredictorParams extends Params
     validateParams()
     // TODO: Support casting Array[Double] and Array[Float] to Vector when FeaturesType = Vector
     SchemaUtils.checkColumnType(schema, $(featuresCol), featuresDataType)
-    if (fitting) {
-      // TODO: Allow other numeric types
-      SchemaUtils.checkColumnType(schema, $(labelCol), DoubleType)
-    }
+    if (fitting) SchemaUtils.checkNumericType(schema, $(labelCol))
```

Comment:
Spark style prefers to keep the braces format for if statements. Can you change it back please?